### ISSUE TITLE
Investigate why some issues were not published

### ISSUE THREAD

**Issue by marco-c** [mozilla/code-review#359]
The job run in https://treeherder.mozilla.org/#/jobs?repo=try&revision=43f38e6e54b966e7898690925773f3c9190355fb&selectedJob=280925582,but we didn't publish a comment on https://phabricator.services.mozilla.com/D56976.
The patch was later backed-out because of the linting failure: https://treeherder.mozilla.org/#/jobs?repo=autoland&searchStr=lint&tochange=dcab270db63af6dfb9ab0e3740c06866c49cb993&fromchange=d2b9b4e977a779e3fa9fe3ccf97bac7ac07afbcd.

**Comment by marco-c** [comment]
Similar situation, but for android-lints in https://phabricator.services.mozilla.com/D56994. The first build failed because we couldn't apply the patch: https://phabricator.services.mozilla.com/B145794. Then the developer set the right dependencies, and the following build succeeded: https://phabricator.services.mozilla.com/B147448.
The bot didn't publish anything, but then the patch was later backed out because of linting issues: https://treeherder.mozilla.org/#/jobs?repo=autoland&tochange=08056399677f857d6f2b8189532cb28cde666a81&searchStr=lint&fromchange=1a9be6d049a78ad37c99298dfce43fba6dd45687&selectedJob=281640508.

**Comment by La0** [comment]
Regarding D56976, [the error was read by the bot, and published on the backend](https://code-review.moz.tools/#/diff/206945), but was not published on Phabricator ([silent issue in the log](https://firefox-ci-tc.services.mozilla.com/tasks/Rg_S4ZrNQL-78KH999YMeg/runs/0/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FRg_S4ZrNQL-78KH999YMeg%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L40) as it's not in the patch)

**Comment by La0** [comment]
Regarding D56994, there are no issues [on the treeherder job](https://treeherder.mozilla.org/#/jobs?repo=try&revision=b2e21d0af8504617c7d30e1b3a4561080da4af8f), no issues were [reported in the backend](https://code-review.moz.tools/#/?search=56994)

The error mentions it can be a merge error ? The same lints job was run on the diffs processed by the bot and reported nothing :-1: 

```
TEST-UNEXPECTED-ERROR | /mobile/android/geckoview/src/main/java/org/mozilla/geckoview/GeckoSession.java:5235 | Constants `PermissionDelegate.PERMISSION_AUTOPLAY_INAUDIBLE` and `PermissionDelegate.PERMISSION_XR` specify the same exact value (3); this is usually a cut & paste or merge error (UniqueConstants)
```

**Comment by marco-c** [comment]
> Edit: also it was marked as new for revision, would surely be marked as new in the repo with before/after

We could already start publishing all issues which are new in the revision, it would be a relatively easy win.

Or we could use the concept of unignorable issues (see also #25): issues coming from jobs which are known to be always green must be new issues (as we know "before" is empty).

**Comment by marco-c** [comment]
> The error mentions it can be a merge error ? The same lints job was run on the diffs processed by the bot and reported nothing -1

Looking at the job on autoland, it looks like the cause of the failure is the last patch, which would be https://phabricator.services.mozilla.com/D57134. Here too there was nothing reported by the bot for https://phabricator.services.mozilla.com/B147449.

**Comment by La0** [comment]
We should do both:
1. report new for revision issues #363 
2. support never broken analyzers #364

**Comment by marco-c** [comment]
Merged into #360.
